maliput_ros_interfaces/srv/SampleLaneSRoute.srv

```diff
@@ -0,0 +1,6 @@
+# Equivalent to call maliput::api::RoadNetwork::road_geometry()->SampleAheadWaypoints(lane_s_route, path_length_sampling_rate)
+# and return the vector of maliput::api::InertialPositions.
+LaneSRoute lane_s_route
+float64 path_length_sampling_rate
+---
+InertialPosition[] waypoints
```

Comment:
Wondering if we should offer a service asking for a lanesroute but it automatically goes over the SampleAheadWaypoints method. Like a DeriveLaneSRoutesWaypoints srv. In order to avoid the user concatenating two services and having two waits.

Reply:
DeriveLaneSRoutesWaypoints may not be possible because DeriveLaneSRoutes retrieves a vector of LaneSRoutes, each one is a possible path between the start and the end road positions.
We may want to filter by rules or lane constraints first before sampling.

Reply:
Ah you're right